src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -11,9 +11,10 @@ import { YourDetails } from 'response/tasks/yourDetails'
 import { FreeMediationTask } from 'response/tasks/freeMediationTask'
 import { Claim } from 'claims/models/claim'
 import { WhenDidYouPayTask } from 'response/tasks/whenDidYouPayTask'
+import { isPastResponseDeadline } from 'claims/isPastResponseDeadline'
 
 export class TaskListBuilder {
-  static buildBeforeYouStartSection (draft: ResponseDraft, claim: Claim): TaskList {
+  static buildBeforeYouStartSection (draft: ResponseDraft, claim: Claim, now: moment.Moment): TaskList {
```

Comment:
why is this being passed in? Should it not just be set inside the method?

Reply:
Without it testing becomes very hard.

Reply:
fair enough